main.go

```diff
@@ -65,13 +81,28 @@ func main() {
 
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
+	err := cfg.Validate()
+	if err != nil {
+		setupLog.Error(err, "error in driver configuration")
+		os.Exit(1)
+	}
+
+	if leaderElectionNamespace == "" {
+		fmt.Println("leader-election-namespace is empty")
```

Comment:
Should/Can this be logged as well?

Reply:
I tried to log initially it becomes messy as we need to define a new error variable and also `setupLog.Error` Needs two arguments.

Reply:
yes, but with this approach we lose log when we exit. is there a way to debug why we exited?
If yes, then I won't insist on logging.

Reply:
done now logging to stderr